### ISSUE TITLE
Model limitation

### ISSUE THREAD

**Issue by qhlx** [gusye1234/nano-graphrag#84]
openai.NotFoundError: Error code: 404 - {'error': {'message': 'The model `gpt-4o` does not exist or you do not have access to it.', 'type': 'invalid_request_error', 'param': None, 'code': 'model_not_found'}}
I use the example code, but it seems that I can't use get-4o.

**Comment by rangehow** [comment]
Which file are you running?

**Comment by qhlx** [comment]
<img width="855" alt="截屏2024-10-22 15 43 16" src="https://github.com/user-attachments/assets/1c151906-e90a-431e-99a9-ca396a229498">
this one，thanks

**Comment by qhlx** [comment]
I think the problem is that got-4o is not free, so I can't use it.

**Comment by rangehow** [comment]
Yes, openai api requires payment. You may need to set API_KEY before requesting it. Or you can try local llm following instruction in  /example

**Comment by qhlx** [comment]
Sorry, I meet another question. I run the no_openai_key_at_all.py.
<img width="1283" alt="截屏2024-10-22 17 54 25" src="https://github.com/user-attachments/assets/fd3fc47a-44d5-41c1-802d-e092e2e351d3">

**Comment by qhlx** [comment]
I run the qwen2.5, and it return this consequence in the console, is that correct?
![Uploading 截屏2024-10-23 12.05.41.png…]()

**Comment by gusye1234** [comment]
Sure
